python/ray/air/_internal/device_manager/__init__.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from typing import Optional, Type
+
+import ray
+import ray._private.ray_constants as ray_constants
+from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
+from ray.air._internal.device_manager.cpu import CPUTorchDeviceManager
+from ray.air._internal.device_manager.hpu import HPUTorchDeviceManager
+from ray.air._internal.device_manager.npu import (
+    NPU_TORCH_PACKAGE_AVAILABLE,
+    NPUTorchDeviceManager,
+)
+from ray.air._internal.device_manager.nvidia_gpu import CUDATorchDeviceManager
+from ray.air._internal.device_manager.torch_device_manager import TorchDeviceManager
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_TORCH_DEVICE_MANAGER_CLS = CPUTorchDeviceManager
+
+
+SUPPORTED_ACCELERATOR_TORCH_DEVICE_MANAGER = {
+    ray_constants.GPU: CUDATorchDeviceManager,
+    ray_constants.HPU: HPUTorchDeviceManager,
+    ray_constants.NPU: NPUTorchDeviceManager,
+}
+
+
+def try_register_torch_accelerator_module(backend=None) -> None:
+    try:
+        if HPU_PACKAGE_AVAILABLE:
+            import habana_frameworks.torch.hpu  # noqa: F401
+
+            if backend == "hccl":
+                import habana_frameworks.torch.core  # noqa: F401
+                import habana_frameworks.torch.distributed.hccl  # noqa: F401
+
+        if NPU_TORCH_PACKAGE_AVAILABLE:
+            import torch_npu  # noqa: F401
```

Comment:
nit: shard these into individual methods in each `device_manager/<accelerator>.py` file